rf/harmonics.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2013-2019 Tom Eulenfeld, MIT license
+"""
+Harmonic decomposition
+"""
+from copy import copy
+import numpy as np
+import warnings
+from rf.util import _add_processing_info
+
+@_add_processing_info
+def harmonics(stream, first_comp='R',second_comp=None, azim=0, method='time', **kwargs):
```

Comment:
You could use a single argument `components='RR'` or similar instead of the two arguments. The logic of the function could be the same after `same_components = componetns[0] == components[1]` or similar.